framework/MessageHandler.py

```diff
@@ -174,7 +174,7 @@ def __init__(self):
     self.suppressErrs = False
     self.printTime    = True
     self.inColor      = False
-    self.verbCode     = {'silent':0, 'quiet':1, 'all':2, 'debug':3}
+    self.verbCode     = {'silent':0, 'quiet':1, 'all':2, 'debug':3, 'none':2}
```

Comment:
none == 2?

Reply:
Yes, default is 2, and while I don't recall why at this point, I was getting the string 'none' for a value at one point when one wasn't provided.

Reply:
This showed up when using the message handler for an external entity. For now I'm removing the option and forcing the external entity to set its own verbosity rather than accept None as a default.